dietpi/dietpi-software

```diff
@@ -1837,6 +1837,19 @@ DietPi-Software will decrypt and use it for software installs. You can change it
 		aSOFTWARE_REQUIRES_FFMPEG[$software_id]=1
 		aSOFTWARE_ONLINEDOC_URL[$software_id]='p=70#p70'
 
+		#------------------
+		software_id=177
```

Comment:
Software IDs 27 and 140 are now available as well, so no need to increase the maximum number currently 😃.
```
root@VM-Buster:~# dietpi-software free
[  OK  ] DietPi-Software | Initialised database
[  OK  ] DietPi-Software | Reading database
Free software ID(s): 27 140
```

Reply:
Will change it to ID 27